### ISSUE TITLE
Quick pick: tab should focus the action, not close Quick pick

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51839]
**Environment Details:**
- VSCode Version:1.24
- OS Version:Windows10
**Additional Details:**
Keyboard,[,MAS2.1.1]
**Steps to Reproduce:**
1.Launch VSCode.
2.Navigate   to Go(Alt+G) menu option and select "Go To File" from dropdown.
3.Use the "Tab key" to navigate "Recently viewed" icon.
**Actual:**
keyboard focus does not go to "Recently viewed" icon.
NVDA Also does not read the "Recently viewed".
**Expected:**
keyboard focus should go to "Recently viewed" icon.
NVDA should read the "Recently viewed".
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](url)
**[User] Impact:**
User will not be able to use the control.
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:**
![accessibility a11y_vscode_gotofile_keyboard](https://user-images.githubusercontent.com/39204909/41395539-b72ef84c-6fcb-11e8-9bfa-c7c3365f3902.png)

[vedioGOTOFILE.pptx](https://github.com/Microsoft/vscode/files/2101669/vedioGOTOFILE.pptx)

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51850
